src/lib/style-transform.js

```diff
@@ -152,16 +97,15 @@ function transform(hash, styles, settings = {}) {
   generator = settings.generator
   offset = settings.offset
   filename = settings.filename
-  isSplitRulesEnabled = settings.splitRules
   splitRules = []
-  scopeHash = hash
 
-  const cssString = stylis(scopeHash, styles)
+  stylis(hash, styles)
 
-  if (isSplitRulesEnabled) {
+  if (settings.splitRules) {
     return splitRules
   }
-  return cssString
+
+  return splitRules.join('')
```

Comment:
@thysultan as you can see from the diff before I was returning an array of `splitRules` only when `settings.splitRules` was set to `true` otherwise we'd just return the processed string. With stylis-rule-sheet the string contains `/*|*/` therefore I decided to return `splitRules.join('')` - do you see any cons of doing so?